crates/rolldown_rstr/src/lib.rs

```diff
@@ -4,21 +4,22 @@
 
 use std::{fmt::Display, ops::Deref};
 
-use oxc::span::{Atom, CompactString};
+use oxc::span::{Atom, CompactStr};
 
 mod to_str;
 pub use to_str::ToRstr;
 
-#[derive(Debug, Clone, Hash, PartialEq, Eq, Default)]
-pub struct Rstr(CompactString);
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+pub struct Rstr(CompactStr);
 
 impl Rstr {
   pub fn as_str(&self) -> &str {
     self.0.as_str()
   }
 
   pub fn to_oxc_atom(&self) -> Atom<'static> {
```

Comment:
The right way to do here is remove `to_oxc_atom` method and create `Atom` in arena. This will cause a lot changes, you could leave me handle it.